tests/unit/s2n_ktls_test.c

```diff
@@ -38,5 +75,163 @@ int main(int argc, char **argv)
         EXPECT_FALSE(cipher.ktls_supported);
     };
 
+    if (!s2n_ktls_is_supported_on_platform()) {
+        DEFER_CLEANUP(struct s2n_connection *server_conn = s2n_connection_new(S2N_SERVER),
+                s2n_connection_ptr_free);
+        int fd = 1;
+        EXPECT_OK(s2n_test_configure_mock_ktls_connection(server_conn, fd, true));
+
+        EXPECT_FAILURE_WITH_ERRNO(s2n_connection_ktls_enable_send(server_conn), S2N_ERR_KTLS_UNSUPPORTED_PLATFORM);
+        EXPECT_FAILURE_WITH_ERRNO(s2n_connection_ktls_enable_recv(server_conn), S2N_ERR_KTLS_UNSUPPORTED_PLATFORM);
+    } else {
+        /* ktls handshake must be complete */
+        {
+            DEFER_CLEANUP(struct s2n_connection *server_conn = s2n_connection_new(S2N_SERVER),
+                    s2n_connection_ptr_free);
+            int fd = 1;
+            EXPECT_OK(s2n_test_configure_mock_ktls_connection(server_conn, fd, false));
+
+            EXPECT_FAILURE_WITH_ERRNO(s2n_connection_ktls_enable_send(server_conn), S2N_ERR_HANDSHAKE_NOT_COMPLETE);
+            EXPECT_FAILURE_WITH_ERRNO(s2n_connection_ktls_enable_recv(server_conn), S2N_ERR_HANDSHAKE_NOT_COMPLETE);
+        };
+
+        /* s2n_connection_ktls_enable */
```

Comment:
Can you take another look at your test naming? All these tests test s2n_connection_ktls_enable_send / s2n_connection_ktls_enable_recv.

Generally we structure tests like:
```
/* Test s2n_connection_ktls_enable_send/recv */
{

    /* Fails if TLS handshake not complete */
   {
   };

   /* Noop if ktls already enabled */
   {
   };
};
```

I imagine you plan to add other methods to s2n_ktls.c, and you'll want to be clear on what you're testing.